modules/Bio/EnsEMBL/Compara/RunnableDB/Ortheus.pm

```diff
@@ -607,7 +608,7 @@ sub parse_results {
 
 			print "start_X $start_X end_X $end_X subseq_length " . length($subseq) . "\n" if ($self->debug);
 
-			$genomic_align->cigar_line($start_X . "X" .$genomic_align->cigar_line . $end_X . "X");
+			$genomic_align->cigar_line( Bio::EnsEMBL::Compara::Utils::Cigars::pad_with_x($genomic_align->cigar_line, $start_X, $end_X) );
```

Comment:
```suggestion
                        $genomic_align->cigar_line( Bio::EnsEMBL::Compara::Utils::Cigars::pad_with_x($genomic_align->cigar_line, $start_X, $end_X) );
```
I think I have replaced the tabs by the right number of spaces.

Reply:
I would actually leave the tabs as they are as they fit with the rest of the function. But I would change the other occurrence in Ortheus.pm

Reply:
I'm also going to restore the indentation in RunnableDB/EpoLowCoverage/LowCoverageGenomeAlignment.pm